internal/backends/linuxkms/calloop_backend/input.rs

```diff
@@ -218,14 +226,21 @@ impl<'a> calloop::EventSource for LibInputHandler<'a> {
                         {
                             i_slint_core::api::quit_event_loop()
                                 .expect("Unable to quit event loop multiple times");
```

Comment:
I suppose that at this point we may want to return, or at least make sure to to go to they key dispatching further down.